StringCalculator.Core.Tests/Processor/StringCalculatorProcessorTest.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using StringCalculator;
+
+namespace StringCalculator.Core.Processor
+{
+    public class StringCalculatorProcessorTest
+    {
+        private StringCalculatorProcessor calculator;
```

Comment:
is better to initiate the calculator in Constructor here instead of init in each method